app/config.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+class Config:
+    MONGO_DBNAME = "2BN-Desserts"
+    MONGO_URI = os.getenv("MONGO_URI")
+    SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
Add an extra blank line at the end of every file

Reply:
I've done that now to all files (x28). Do you know the reasoning as to why this is required? I've never heard of it. Thanks for that!